README.md

```diff
@@ -18,8 +18,8 @@ ACCESS-ESM1.6 is built and deployed automatically to `gadi` on NCI (see below).
 Then clone this repository and run the following commands on `gadi`:
 
 ```bash
-spack env create access-esm1.6 spack.yaml
-spack env activate access-esm1.6
+spack env create access-esm1p6 spack.yaml
+spack env activate access-esm1p6
```

Comment:
Should we encourage the use of `spack env activate -p` such that the environment is listed in `PS1`?

Reply:
Not sure. Any opinion @harshula?

Reply:
I like `-p`.

Reply:
Added in https://github.com/ACCESS-NRI/ACCESS-ESM1.6/pull/108/commits/09af6719b220adcb4246f310b60e0ae9e097b75e